dev/building.md

```diff
@@ -220,6 +220,29 @@ This can however hide include bugs, this flag disables using the unity build so
 In some situations, running an executable that has been built with sanitizers enabled is not support / can cause problems. Julia is an example of this.  
 With this flag enabled, the sanitizers are disabled for the build.
 
+## Dependencies of the Python Library on Windows
+
+The DuckDB dynamic library used on Windows (`.pyd`) has the following dependencies when built with Python 3.10:
```

Comment:
Listing out each ddl isn't really useful, just say that the visual C++ redistributable is required